src/vs/workbench/contrib/webview/browser/themeing.ts

```diff
@@ -73,6 +74,7 @@ export class WebviewThemeDataProvider extends Disposable {
 				'vscode-editor-font-family': editorFontFamily,
 				'vscode-editor-font-weight': editorFontWeight,
 				'vscode-editor-font-size': editorFontSize + 'px',
+				'vscode-editor-font-feature-settings': editorFontLigatures,
```

Comment:
```suggestion
				'vscode-editor-font-ligatures': editorFontLigatures,
```

Let's match the VS Code setting name just to make it very clear this comes from that setting